### ISSUE TITLE
🐛[BUG][System][Combat] Crystal Drop rates recently reduced after recent patch

### ISSUE THREAD

**Issue by coding4saudade** [HorizonFFXI/HorizonXI-Issues#780]
**What does this bug affect?**

Since the patch this morning 1/9/2023, when trying to farm crystals, the drop rates are now super low. Drop rates were close to 1 in 3. Now it is more like 1 in 30.
Players who are trying to start crafting are having difficulty getting started. For players with lot of gil, this is a nonissue. For those who are poor, being unable to viably farm crystals is a roadblock to getting into crafting.  For some, crystal farming is the main way they make gil.  

**Steps to Reproduce**
Get signet, kill mobs. tested at level 1-40 so far.  In regions owned by home nation or not.  Test on mobs not close to home nations in case those mobs have different drop parameters.

**Expected behavior**
I would expect crystals to drop with a higher rate.  1 in 3 or 4 instead of 1 in 30

**Comment by coding4saudade** [comment]
https://discord.com/channels/933423693848260678/1060984980739735644

**Comment by Atigeve** [comment]
I can also confirm this I spent several hours testing in different areas (Ghelsba Outpost and Meriph Mountaints and Konschtat Highlands) at one point in Meriph Mountains I went nearly 30 mins with signet on without a single crystal drop fighting EP constantly.  Ghelsba Outpost/Fort Ghelsba I spent 2-3 hours on a level 10 job (got it to lvl 13 on EP there) trying to farm fire crystals in that period I got a total of 7 or 8 crystals.  Konschtat was the same thing, fighting mist lizards for both skins and crystals, very very few crystals and 0 skins (made a bug report on the skins) and that was like 3-5 hours there.
